README.md

```diff
@@ -3,21 +3,132 @@
+- OIDC Client Name: the OIDC client name that should be used by MLSpace for authentication
+
+If selecting Advanced Config you will be prompted for the same properties Basic Config prompts for, as well as other optional values. Anything not specific will use the defaults in `constants.ts` and/or provisioned by MLSpace. 
+
+The Advanced Config will ask:
+
+**Do you want to use existing VPC?**
+If you answered yes you will be prompted for:
+
+- VPC Name: if MLSpace is being deployed into an existing VPC this should be the name of that VPC
+- VPC ID: if MLSpace is being deployed into an existing VPC this should be the ID of that VPC
+- VPC Default Security Group: if MLSpace is being deployed into an existing VPC this should be the default security group of that VPC
+
+**Do you want to use existing IAM Roles?**
+If you answered yes you will be prompted for:
+
+- S3 Reader Role ARN: ARN of an existing IAM role to use for reading from the static website S3 bucket
+- Bucket Deployment Role ARN: ARN of an existing IAM role to use for deploying to the static website S3 bucket
+- Notebook Role ARN: ARN of an existing IAM role to associate with all notebooks created in MLSpace
+- App Role ARN: ARN of an existing IAM role to use for executing the MLSpace lambdas
+- EMR Default Role ARN: ARN of an existing IAM role that will be used as the 'ServiceRole' for all EMR clusters
+- EMR EC2 Instance Role ARN: ARN of an existing role that will be used as the 'JobFlowRole' and 'AutoScalingRole' for all EMR clusters
+
+**Do you want to modify the banner displayed on MLSpace?**
+If you answered yes you will be prompted for:
+
+- System Banner Text: the text to display on the system banner displayed at the top and bottom of the MLSpace web application. If set to a blank string no banner will be displayed
+- System Banner Background Color: the background color of the system banner if enabled
+- System Banner Text Color: the color of the text displayed in the system banner if enabled
+
+### Option 2 - configure by updating lib/constants.ts
+
+Configure MLSpace using Option 2 if:
+
+- the MLSpace Config Wizard doesn't configure all of the settings you need to customize
+- you wish to have your configuration changes in a file that's committed to git
+
+If you are pre-creating roles you will need to ensure that the required role arns (`APP_ROLE_ARN`, `NOTEBOOK_ROLE_ARN`), role names (`KEY_MANAGER_ROLE_NAME` if `EXISTING_KMS_MASTER_KEY_ARN` is not set), and `AWS_ACCOUNT` (used to ensure unique S3 bucket names) have been properly set in `lib/constants.ts`.
+
+You will also need to set `OIDC_URL` and `OIDC_CLIENT_NAME` with the correct values based on your chosen IdP. These property must be set prior to deploying MLSpace.
+
+To see the full list of configurable properties and their descriptions, see the [Configurable deployment parameters section](Configurable deployment parameters).
+
+### Creating a production optimized web app build
```

Comment:
Should we add getting dependencies for the backend? I have noticed this lacking in the documentation. Does this work if you do it not in a dev container with only the steps listed in the README? Note for another PR